src/player.cpp

```diff
@@ -11448,7 +11448,10 @@ hint_rating player::rate_action_read(item *it)
 
     return HINT_GOOD;
 }
-
+void player::acquire_xp(int amount){
```

Comment:
Ah, meant to remove that, just wanted to have a way to test the function. I'll get rid of it.